now-staging.json

```
@@ -0,0 +1,12 @@
+{
+    "version": 2,
+    "public": true,
+    "build": {
+      "env": {
+        "REACT_APP_CHAIN_ID": "4",
+        "REACT_APP_APP_NAME":"disputable-conviction-voting",
+        "REACT_APP_APP_RINKEBY_INSTANCE": "staging"
```

What do you think of making it more general and not network sepcific? 
```suggestion
        "REACT_APP_APP_INSTANCE": "staging"
```